lago/cmd.py

```diff
@@ -277,6 +277,25 @@ def do_stop(prefix, vm_names, **kwargs):
     prefix.stop(vm_names=vm_names)
 
 
+@lago.plugins.cli.cli_plugin(help='Shutdown and destroy lago resources')
+@lago.plugins.cli.cli_plugin_add_argument(
+    '-r',
+    '--reboot',
```

Comment:
So the plugin help says 'Shutdown and destroy' - how does that work with reboot?